Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.Be.cs

```diff
@@ -231,11 +231,71 @@ public void When_text_is_longer_than_8_characters_only_add_ellipsis_where_applic
                 "Expected subject to be the same string because we use arrows now, but they differ at index 5:" +
                 Environment.NewLine
                 + "        ↓ (actual)" + Environment.NewLine
-                + "  \"this is a long text…\"" + Environment.NewLine
+                + "  \"this is a long text that…\"" + Environment.NewLine
                 + "  \"this was too short\"" + Environment.NewLine
                 + "        ↑ (expected).");
         }
 
```

Comment:
What about adding the expected message as raw string?

like:
```csharp

string msg = """
             Expected subject to be the same string because we use arrows now, but they differ at index 5:
                    ↓ (actual)
              "this is a long text that…"
              "this was too short"
                    ↑ (expected).
             """;
```

This allows to read quickly through the tests and see where the pointers are pointing to. Now I have to stop an "think the `\` away" in my mind to get the actual position of the arrows.

Reply:
I just did this with [6cbf802](https://github.com/fluentassertions/fluentassertions/pull/2307/commits/6cbf80244606b611ac14bfb1db1b1fc746b219d3) :-)

Reply:
Oh.. I see.. 🎉